www/source/docs/reference/environment-vars.html.md

```diff
@@ -25,6 +25,7 @@ This is a list of all environment variables that can be used to modify the opera
 | `HAB_STUDIO_SUP` | build system | no default | Used to customize the arguments passed to an automatically launched Supervisor, or to disable the automatic launching by setting it to `false`, `no`, or `0`. |
 | `HAB_UPDATE_STRATEGY_FREQUENCY_MS` | supervisor | 60000 | Frequency of milliseconds to check for updates when running with an [update strategy](/docs/run-packages-update-strategy) |
 | `HAB_USER` | supervisor | no default | User key to use when running with [service group encryption](/docs/run-packages-security/#service-group-encryption) |
+| `HAB_NON_ROOT` | supervisor | no default | Enable the supervisor to install and run packages as non-root user. *Warning* it is highly reckomended not to use this setting except under very specific circumstances due to associated security risks. |
```

Comment:
Spelling: "reckomended" -> "recommended"

Adding an example of when using this variable *would* be appropriate would help users figure out if they should be using it or not.

Also, doesn't this environment variable just govern installation of packages, rather than installation *and* running as a non-root user?

Reply:
Thanks for this. I'll add an example when our cf-exporter gets merged. Then the example is more concrete and less hypothetical